pkg/bootstrap/bootstrap.go

```diff
@@ -132,6 +138,14 @@ func (c *BootstrapClient) Bootstrap(
 		manifests = append(manifests, defaultRepository())
 	}
 
+	if options.DryRun {
+		statusMessage("Dry-run mode enabled. The following resources would be applied:", true)
+		for _, manifest := range manifests {
```

Comment:
I think printing the manifest names here is not necessary, and also it would make the output yaml invalid this way.